### PR TITLE
Pensions spouse marriage changes

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3486

Updates marriages and spouse marriage pages with feedback from stakeholders.

screen shot 2017-06-25 at 3 22 44 pm
screencapture-localhost-3001-pension-application-527ez-household-spouse-marriages-0-1498418601206

### CODE REVIEW

File: src/js/common/schemaform/widgets/ArrayCountWidget.jsx [hunk 1]
```diff
@@ -31,7 +31,12 @@ export default class ArrayCountWidget extends React.Component {
   }
 
   updateArrayLength = (event) => {
-    const count = event.target.value;
+    let count = event.target.value;
+    // You can lock up the browser by setting too high of a number here
+    // and 29 appears to be the record, so this seems safe
```
Comment (cehsu) [inline src/js/common/schemaform/widgets/ArrayCountWidget.jsx]: [Reference for this](https://en.wikipedia.org/wiki/Glynn_Wolfe)

File: src/js/pensions/config/form.js [hunk 2]
```diff
@@ -755,6 +750,10 @@ const formConfig = {
                 reasonForSeparation: {
                   'ui:title': 'Why did the marriage end?',
                   'ui:widget': 'radio'
+                },
+                dateOfSeparation: currentOrPastDateUI('Date marriage ended'),
+                locationOfSeparation: {
+                  'ui:title': 'Place marriage ended',
```
Comment (cvalarida) [inline src/js/pensions/config/form.js]: I'm wondering if this makes sense / should be required if the applicant is widowed. @J-Quag?

**Review by cvalarida — APPROVED** [review]
Just not sure if having the date and location of marriage ending makes sense to have (or require) if they're widowed.

**Review by cehsu — APPROVED** [review]

File: src/js/pensions/helpers.jsx [hunk 3]
```diff
@@ -33,8 +37,14 @@ export function getMarriageTitle(index) {
   return desc ? `${desc} marriage` : `Marriage ${index + 1}`;
 }
 
+export function getSpouseMarriageTitle(index) {
+  const desc = numberToWords[index];
+
+  return desc ? `Spouse’s ${desc.toLowerCase()} marriage` : `Spouse marriage ${index + 1}`;
```
Comment (cehsu) [inline src/js/pensions/helpers.jsx]: Was the decision to use the word spouse or the spouse's name here?

File: src/js/pensions/config/form.js (hunk 2 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: The paper form doesn't make any distinction and based on the walkthrough, I think these fields were expected for widows.

**Pull request merged**